modules/util/sec_to_time.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package util
+
+import "fmt"
+
+// SecToTime converts an amount of seconds to a human-readable string (example: 66s -> 1min 6s)
+func SecToTime(duration int64) string {
+	seconds := duration % 60
+	minutes := (duration / (60)) % 60
+	hours := duration / (60 * 60) % 24
+	days := duration / (60 * 60) / 24
+
+	var hrs string
```

Comment:
These doesn't involve just hrs anymore, I think a more appropriate variable name can be used like `formattedTime`